mobile_verifier/src/cell_type.rs

```diff
@@ -49,8 +49,21 @@ impl CellType {
             Self::Neutrino430 => CellTypeLabel::CBRS,
             Self::SercommIndoor => CellTypeLabel::CBRS,
             Self::SercommOutdoor => CellTypeLabel::CBRS,
+            Self::NovaGenericWifi => CellTypeLabel::Wifi,
             Self::CellTypeNone => CellTypeLabel::CellTypeLabelNone,
-            Self::NovaGenericWifiIndoor => CellTypeLabel::Wifi,
+        }
+    }
+
+    pub fn to_proto(&self, indoor: bool) -> CellTypeProto {
+        match self {
+            Self::Nova436H => CellTypeProto::Nova436h,
+            Self::Nova430I => CellTypeProto::Nova430i,
+            Self::Neutrino430 => CellTypeProto::Neutrino430,
+            Self::SercommIndoor => CellTypeProto::SercommIndoor,
+            Self::SercommOutdoor => CellTypeProto::SercommOutdoor,
+            Self::NovaGenericWifi if indoor => CellTypeProto::NovaGenericWifiIndoor,
```

Comment:
Why have only a single `NovaGenericWifi` covering both indoor and outdoor for the internal struct but distinct values on the proto struct ?  I suspect its because we dont want to change the proto as its already out in the wild ?  

In general I dont really see the benefit in the change  to a single value, especially if the external representations use distinct values ( ie the protos and the onchain representations ).  It also requires any code which needs to distinguish between indoor and outdoor to rely on the coverage object being present/available...that may be a safe assumption ( not entirely sure yet ) but doesnt seem ideal.

Reply:
I think that's reasonable, I'll change around the order

Reply:
I am not sure I understand why we care about cell_type anymore.  Isn't it just cbrs or wifi and indoor or outdoor?

Reply:
We need it for our proto output.
I think it generally cleans up the implementation so I'm fine doing it.